invenio_rdm_records/theme/static/images/orcid.svg

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 width="255.999px" height="256px" viewBox="0 0 255.999 256" enable-background="new 0 0 255.999 256" xml:space="preserve">
+<path fill="#A6CE39" d="M255.999,128.008C255.999,198.697,198.69,256,127.997,256C57.306,256,0,198.697,0,128.008
+	S57.306,0,127.997,0C198.69,0,255.999,57.318,255.999,128.008z"/>
+<g>
+	<path fill="#FFFFFF" d="M86.337,186.215H70.943V79.102h15.394v48.387V186.215z"/>
+	<path fill="#FFFFFF" d="M108.879,79.102h41.624c39.578,0,57,28.271,57,53.559c0,27.486-21.506,53.555-56.844,53.555h-41.78V79.102z
+		 M124.283,172.379h24.495c34.859,0,42.867-26.531,42.867-39.719c0-21.527-13.66-39.746-43.652-39.746h-23.71V172.379z"/>
+	<path fill="#FFFFFF" d="M88.728,56.811c0,5.541-4.538,10.096-10.086,10.096c-5.555,0-10.094-4.555-10.094-10.096
+		c0-5.555,4.539-10.102,10.094-10.102C84.189,46.709,88.728,51.256,88.728,56.811z"/>
+</g>
+</svg>

```

Comment:
Both png and svg are needed?

Reply:
You are right. For now we are only using the png, which is less scalable. I change the usage to svg and removed the png file :).